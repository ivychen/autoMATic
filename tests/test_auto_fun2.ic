auto print_str(string s) {
    printstr(s);
}

auto main() {
    auto str = "test string";
    print_str(str);
    print(print_num(360));

    return 0;
}

auto print_num(int x) {
    print(x);
    return 50;
}
