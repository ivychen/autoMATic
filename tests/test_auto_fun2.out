test string
360
50
